http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/package.scala

```diff
@@ -1,11 +1,10 @@
-package org.broadinstitute.dsde.workbench
+package org.broadinstitute.dsde.workbench.leonardo
 
 import org.broadinstitute.dsde.workbench.leonardo.db.DBIOOps
 import org.broadinstitute.dsde.workbench.model.ErrorReportSource
 import slick.dbio.DBIO
 
-package object leonardo {
+package object http {
```

Comment:
This means it'll be available to everything in `http`?

Reply:
yep...everything in `core` is directly under `leonardo` namespace